splitio/go-client
# Feature/ip address enabled flag

# GO SDK

## What did you accomplish?
- Added `IPAddressesEnabled` to enable/disable sending MachineName and MachineIP headers when data is posted to Split Servers.

## How do we test the changes introduced in this PR?
- `go test ./...`
- When posting data you will see that the header is not sent when the config is disabled.
- You can check with ImpressionListener. `InstanceName` should have `NA` value.

## Extra Notes
